nextcloudappstore/core/models.py

```diff
@@ -118,6 +159,12 @@ def can_delete(self, user: User) -> bool:
     def __str__(self) -> str:
         return '%s %s' % (self.app, self.version)
 
+    def is_compatible(self, platform_version):
+        platform_version = Version(platform_version)
```

Comment:
for platform_version we want to use the pad_min_version function here https://github.com/nextcloud/appstore/blob/master/nextcloudappstore/core/versioning.py#L25

The reasoning behind this is that a user does not want to see Nextcloud patch versions when downloading archives (e.g. he does not want 9.0.0, he wants 9.0).